pom.xml

```diff
@@ -44,7 +44,7 @@
         <dependency>
             <groupId>org.hobbit</groupId>
             <artifactId>core</artifactId>
-            <version>1.0.6-SNAPSHOT</version>
+            <version>1.0.8-SNAPSHOT</version>

```

Comment:
We are releasing a new version so we don't want to have `-SNAPSHOT` there.
However, after this pull request, the develop branch should be increased to `1.0.9-SNAPSHOT`

Reply:
@gsjunior86 could you please fix this as well?